web_auto_refresh/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+{
+    'name': 'Auto Refresh',
+    'version': '10.0.1.0.0',
+    'author': 'Fisher Yu, Odoo Community Association (OCA)',
+    'category': 'Base',
+    'website': 'https://github.com/szufisher',
```

Comment:
Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)